speechbrain/lobes/models/huggingface_transformers/whisper.py

```diff
@@ -77,20 +89,39 @@ def __init__(
         encoder_only=False,
         freeze=False,
         freeze_encoder=False,
-        output_attentions=True,
+        output_attentions=False,
```

Comment:
In the docstring for this, it should probably be mentioned why someone would want it to be `False` (because of flash attention as far as I understand).